### DISCUSSION TITLE
SM Youtube player either doesn't load or when it does it's sound only

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#949] [General]
Recently  it started doing this. Before it was working OK. Has something changed or is there a setting that might fix it? Thanks.

**Comment by Grooves1210** [comment]
you can try updating yt-dlp.exe

C:\Users\USERNAME\.smplayer
or
PORTABLE_VERSION_FOLDER\mpv

From command prompt at that location:  yt-dlp -U

**Reply by tack32** [reply]
Right now  it's behaving. but thanks for the suggestion in case it starts again.
